pkg/authorization/cache/readonlyclusterpolicybindingcache.go

```diff
@@ -0,0 +1,108 @@
+package cache
```

Comment:
@smarterclayton many file names similar to this - permutations of readonly(cluster)policy(binding)cache. What needs to be in the file name? ReadOnly on the interfaces I think is useful to describe their abilities, necessary in file names?